charts/primary-site/templates/deployments/inbox-listener.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: inbox-listener
+  labels:
+    app: foxglove-inbox-listener
+spec:
+  selector:
+    matchLabels:
+      app: foxglove-inbox-listener
```

Comment:
since we will presumably be deploying these in a namespace that implies foxglove somehow, I wonder if adding "foxglove-" to all the app names is redundant

Reply:
Yea - it might be redundant. Is there a convention around this? Probably this is redundant.